lms/envs/test.py

```diff
@@ -12,22 +12,30 @@
 # We intentionally define lots of variables that aren't used, and
 # want to import all variables from base settings files
 # pylint: disable=wildcard-import, unused-wildcard-import
+from __future__ import absolute_import
+
+import logging
+import os
 from collections import OrderedDict
+from random import choice
+from string import digits, letters, punctuation
+from uuid import uuid4
 
+import openid.oidutil
 from django.utils.translation import ugettext_lazy
-
-from .common import *
-import os
 from path import Path as path
-from uuid import uuid4
+from six.moves import range
 
-from util.db import NoOpMigrationModules
+from openedx.core.djangoapps.plugins import plugin_settings, constants as plugin_constants
 from openedx.core.lib.derived import derive_settings
 from openedx.core.lib.tempdir import mkdtemp_clean
+from util.db import NoOpMigrationModules
```

Comment:
@amitvadhel I think it might be here.  It looks like util is underneath common, so this may need to move below the common import?

Reply:
@feanil I have updated the order of import